### PR TITLE
add personal fullsize pcb hnah108

### PR DESCRIPTION
**Pull request by vuhopkep**
## Description
add personal fullsize pcb hnah108.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/hnah108/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  _QWERTY,
+  _FN
+};
+
+#define LOWER MO(_FN)
+#define RAISE MO(_QWERTY)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_QWERTY] = LAYOUT_all( /* Base */
+    KC_ESC, KC_F1, KC_F2, KC_F3, KC_F4,   KC_F5, KC_F6, KC_F7, KC_F8,   KC_F9, KC_F10, KC_F11, KC_F12,    KC_PSCR, KC_SLCK, KC_PAUS,    KC_MYCM, KC_CALC, KC_MPLY, MO(1),\
```
Comment (fauxpark) [inline keyboards/handwired/hnah108/keymaps/default/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/handwired/hnah108/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      rgblight_increase_val();
+    } else {
+      rgblight_decrease_val();
+    }
+  }
+}
+
+void rgb_matrix_indicators_user(void)
+{
+        if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/handwired/hnah108/keymaps/default/keymap.c]:
```suggestion
        if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

File: keyboards/handwired/hnah108/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        {
+            rgb_matrix_set_color(30, 0x00, 0xFF, 0x00);
+        } else if (IS_LED_ON(host_keyboard_leds(), USB_LED_NUM_LOCK))
```
Comment (fauxpark) [inline keyboards/handwired/hnah108/keymaps/default/keymap.c]:
```suggestion
        } else if (IS_HOST_LED_ON(USB_LED_NUM_LOCK))
```

File: keyboards/handwired/hnah108/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        {
+            rgb_matrix_set_color(30, 0xFF, 0x00, 0x00);
+        } else if (IS_LED_ON(host_keyboard_leds(), USB_LED_SCROLL_LOCK))
```
Comment (fauxpark) [inline keyboards/handwired/hnah108/keymaps/default/keymap.c]:
```suggestion
        } else if (IS_HOST_LED_ON(USB_LED_SCROLL_LOCK))
```

**Comment by vuhopkep** [comment]
thank you for review. 
I'll corect them all

**Pull request merged**